src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -53,6 +53,18 @@ public static ReadOnlyAddressBook getSampleAddressBook() {
         return sampleAb;
     }
 
+    /*
```

Comment:
Similarly, you might want to consider leaving a comment on top of this commented chunk